src/mxarray.jl

```diff
@@ -652,15 +647,22 @@ function jdict(mx::MxArray)
     for i = 1:nf
         fnames[i] = get_fieldname(mx, i)
         pv::Ptr{Void} = ccall(_mx_get_field_bynum, 
-            Ptr{Void}, (Ptr{Void}, mwIndex, Cint),
-            mx.ptr, 0, i-1)
+            Ptr{Void}, (Ptr{Void}, mwIndex, Cint), mx.ptr, 0, i-1)
         fx = MxArray(pv, false)
-        fvals[i] = jvariable(fx)
+        fvals[i] = Any(fx)
     end
     Dict(zip(fnames, fvals))
 end
 
-function jvariable(mx::MxArray)
+function Function(mx::MxArray)
```

Comment:
I added this in `mexjulia` so that MATLAB function handles could be passed to julia and used as callbacks without any special extra marshaling. However, for this definition to do anything meaningful, `MxArray` needs to be made callable. In the context of embedding Julia in MATLAB, this can be achieved via an underlying invocation of `mexCallMATLABWithTrap`, which I provide in `mexjulia`. I very briefly looked into whether there would be a way to do this in the context of interacting with MATLAB via the engine interface, as `MATLAB.jl` does, but didn't immediately see any obvious analogy. In fact, that was the primary motivation for me to fork the data marshaling code from `MATLAB.jl` into `mexjulia`.